src/style.css

```diff
@@ -1 +1,78 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable string-quotes */
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body,
+header,
+nav,
+a,
+ul,
+li {
+  margin: 0;
+  padding: 0;
+}
+
+.header,
+.nav__list {
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header {
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+/* Браузер автоматично додавав до .nav__logo додаткові 4px,
+довелося встановити фіксовану висоту в 40px, щоб пройшло тест,
+ чи можна це виправити? */
+.nav__logo {
+  height: 40px;
+}
+
+.nav__list {
+  list-style-type: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: block;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-style: normal;
```

Comment:
it is default value

Reply:
If I remove font-family, I will get this 
![image](https://user-images.githubusercontent.com/54063988/198104337-0ae3917b-9a26-448d-ac6a-f04672c0099d.png)

I agree, that font-weight and font-style don't make any sense there, but display (I made link-block instead of link-inline element) and fot-family should be there. Or, maybe I wrong?